source/layout.tex

```diff
@@ -58,3 +58,6 @@
 
 % Leave more room for section numbers in TOC
 \cftsetindents{section}{1.5em}{3.0em}
+
+% Allow for substantial empty space at the bottom of a page.
+\addtolength{\topskip}{0pt plus 25pt}
```

Comment:
Can we perhaps look for a memoir way of doing this, rather than modifying the lengths directly? I worry about hard-to-trace interactions if we go behind memoir's back like this.

Reply:
I have reviewed the memoir documentation, section 2 "Laying out the page", and some of the calculations do consider \topskip (called "t" there), but there is no other explicit mention of \topskip.

Reply:
There doesn't seem to be a memoir way of doing this, and we need it when the page otherwise has no stretchable vertical space.

Reply:
Section 3.5 of the [memoir documentation](http://mirrors.ctan.org/macros/latex/contrib/memoir/memman.pdf) might be helpful (e.g., the memoir-specific `\sloppybottom` macro and discussion surrounding it).